index.dd

```diff
@@ -321,8 +321,8 @@ $(DIVC org_list,
 
 $(DIVC org_list_footer,
     <p class="center">
-        $(LINK2 $(ROOT_DIR)orgs-using-d.html, Testimonials And More Organizations)
-        $(LINK2 $(ROOT_DIR)areas-of-d-usage.html, Current D use)
+        $(FA_ICON quote-left) $(LINK2 $(ROOT_DIR)orgs-using-d.html, Testimonials and more organizations)
```

Comment:
Gramatical nitpick: As this is a title of a link and not a sentence, it makes sense to use title case here and not sentence case.

Reply:
Ok reverted this change.